pulp_2to3_migration/app/migrations/0013_fix_file_remote_migration.py

```diff
@@ -0,0 +1,50 @@
+# Generated by Django 2.2.13 on 2020-08:24 19:00
+
+from django.db import migrations, transaction
+from django.utils import timezone
+
+from urllib.parse import urljoin
+
+
+def fix_file_remote(apps, schema_editor):
+    with transaction.atomic():
+        Pulp2Importer = apps.get_model('pulp_2to3_migration', 'Pulp2Importer')
+        Remote = apps.get_model('core', 'Remote')
+        Pulp2LazyCatalog = apps.get_model('pulp_2to3_migration', 'Pulp2LazyCatalog')
+        pulp2isoimporter_qs = Pulp2Importer.objects.filter(pulp2_type_id='iso_importer',
+                                                           is_migrated=True)
+        pulp2lazycatalog_to_update = []
+        remotes_pk_to_delete = []
+        for importer in pulp2isoimporter_qs:
+            pulp2_config = importer.pulp2_config
+            if pulp2_config.get('feed'):
+                pulp2_config['feed'] = urljoin(pulp2_config['feed'], 'PULP_MANIFEST')
```

Comment:
i don't think i have covered this case "To avoid the situation when user is done with file migration and fixed all the remotes themselves, the user is running a migration plan without file plugin in it but we still look at its remotes and try to migrate them. "
I guess the only way to identify that is look at each importer.pulp3_remote.feed and inspect if it has PULP_MANIFEST. other ideas?

Reply:
That makes sense to me, I'm curious if we actually have any users that would be affected by this, do we know of anyone actually using the migration plugin?

They'd need to have

1) migrated some file repos
2) fixed them up
3) never uninstalled the migration plugin even after they were "done" migrating

It feels like a very small number of users, if any at all.

Reply:
I agree, from @jlsherrill  input we can say that there are users who have already migrated some file data, for the rest  we can only speculate

Reply:
I agree that the number of those is zero or very low. 
Moreover we can't be sure that it's always PULP_MANIFEST.
Pulp 3 allows any name, the file just needs to have a specific format.
So, in theory, if we are trying to fix it, we might break some remotes if the filename is different.
Having said that, it's very very unlikely that someone moved fully to pulp 3 AND had synced a file repo with a custom manifest filename.

+1 to not cover that case and leave it out for users to solve, and hopefully there will be zero of those.

Reply:
the case i am outlining will always have pulp_manifest, because these will be the migrated repos from pulp2 which work only with pulp_manifest and if the user has fixed the migrated repo, this migration will simply remove his remote and on re-run will create a new one.
From the discussion i have understood that the odds are very small so we will not cover it.
@goosemania @dralley do you have more comments on the migration before I squash the commits?

Reply:
Ah interesting.
I had different thinking :)
The migrated users can't re-run migration if they have already started using pulp_file. So the only thing we could do is to migrate during pulp_file upgrade or have a separate script.

Regardless, we are not doing it :)

I have no concerns, however I haven't tested this PR yet, I prefer to do that.

Reply:
i think your thinking is correct. users are not supposed to use/modify resources of  the plugin if they have not fully migrated it.
in case they have fully migrated the plugin then as you stated it might be better to leave the fix on users.

Reply:
No concerns